### PR TITLE
[HIVEMALL-192] Fix typos: graphvis -> graphviz

### PR DESCRIPTION
**Pull request by takuti**
## What changes were proposed in this pull request?

Fix crucial typos.

## What type of PR is it?

Improvement

## What is the Jira issue?

https://issues.apache.org/jira/browse/HIVEMALL-192

## Checklist

- [x] Did you apply source code formatter, i.e., `mvn formatter:format`, for your commit?

### CODE REVIEW

File: core/src/main/java/hivemall/smile/tools/TreeExportUDF.java [hunk 1]
```diff
@@ -141,17 +141,17 @@ public String getDisplayString(String[] children) {
...
-        javascript, graphvis;
+        javascript, graphviz;
 
         @Nonnull
         public static OutputType resolve(@Nonnull String name) throws UDFArgumentException {
             if ("js".equalsIgnoreCase(name) || "javascript".equalsIgnoreCase(name)) {
                 return javascript;
-            } else if ("dot".equalsIgnoreCase(name) || "graphvis".equalsIgnoreCase(name)) {
-                return graphvis;
+            } else if ("dot".equalsIgnoreCase(name) || "graphviz".equalsIgnoreCase(name)) {
```
Comment (myui) [inline core/src/main/java/hivemall/smile/tools/TreeExportUDF.java]: better to have `"graphvis".equalsIgnoreCase(name)` for backward compatibility.

Comment (myui) [reply core/src/main/java/hivemall/smile/tools/TreeExportUDF.java]: `if("dot".equalsIgnoreCase(name) || "graphviz".equalsIgnoreCase(name) || "graphvis".equalsIgnoreCase(name))`

Comment (takuti) [reply core/src/main/java/hivemall/smile/tools/TreeExportUDF.java]: c299bd3355ae1e640f12978c76ffb8d5a912d951

**Comment by myui** [comment]
@takuti LGTM! You can merge this PR.

**Pull request closed without merging**